plugin/src/index.ts

```diff
@@ -58,35 +58,49 @@ export async function onBuild({
 The plugin no longer uses this and it should be deleted to avoid conflicts.\n`)
   }
 
-  if (shouldSkipFunctions(cacheDir)) {
-    await deleteFunctions(constants)
-    return
-  }
-  const compiledFunctionsDir = path.join(cacheDir, '/functions')
+  const neededFunctions = await getNeededFunctions(cacheDir)
 
-  await writeFunctions({ constants, netlifyConfig })
+  await deleteFunctions(constants)
```

Comment:
@ascorbic Am I right that this function can run regardless of which functions are needed, because the relevant functions will be recreated in the `writeFunctions` step?

Reply:
Correct